src/QueryEditorForm.tsx

```diff
@@ -76,75 +83,242 @@ export function QueryEditorForm(props: Props) {
-            fetch={fetchRegions}
-            value={queryWithDefaults.connectionArgs.region ?? null}
-            default={props.datasource.defaultRegion}
-            label={selectors.components.ConfigEditor.region.input}
-            data-testid={selectors.components.ConfigEditor.region.wrapper}
-            labelWidth={11}
-            className="width-12"
-          />
-          <ResourceSelector
-            onChange={onChange('catalogs')}
-            fetch={fetchCatalogs}
-            value={queryWithDefaults.connectionArgs.catalog ?? null}
-            default={props.datasource.defaultCatalog}
-            dependencies={[queryWithDefaults.connectionArgs.region]}
-            label={selectors.components.ConfigEditor.catalog.input}
-            data-testid={selectors.components.ConfigEditor.catalog.wrapper}
-            labelWidth={11}
-            className="width-12"
-          />
-          <ResourceSelector
-            onChange={onChange('databases')}
-            fetch={fetchDatabases}
-            value={queryWithDefaults.connectionArgs.database ?? null}
-            default={props.datasource.defaultDatabase}
-            dependencies={[queryWithDefaults.connectionArgs.catalog]}
-            label={selectors.components.ConfigEditor.database.input}
-            data-testid={selectors.components.ConfigEditor.database.wrapper}
-            labelWidth={11}
-            className="width-12"
-          />
-          <ResourceSelector
-            onChange={onChange('tables')}
-            fetch={fetchTables}
-            value={props.query.table || null}
-            dependencies={[queryWithDefaults.connectionArgs.database]}
-            tooltip="Use the selected table with the $__table macro"
-            label={selectors.components.ConfigEditor.table.input}
-            data-testid={selectors.components.ConfigEditor.table.wrapper}
-            labelWidth={11}
-            className="width-12"
-          />
-          <ResourceSelector
-            onChange={onChange('columns')}
-            fetch={fetchColumns}
-            value={props.query.column || null}
-            dependencies={[queryWithDefaults.table]}
-            tooltip="Use the selected column with the $__column macro"
-            label={selectors.components.ConfigEditor.column.input}
-            data-testid={selectors.components.ConfigEditor.column.wrapper}
-            labelWidth={11}
-            className="width-12"
-          />
-          <ResultReuse enabled={resultReuseSupported} query={props.query} onChange={props.onChange} />
-          {!props.hideOptions && (
-            <>
-              <h6>Frames</h6>
-              <FormatSelect query={props.query} options={SelectableFormatOptions} onChange={props.onChange} />
-            </>
-          )}
-        </div>
+      {newFormStylingEnabled ? (
+        <EditorRows>
+          <EditorRow>
+            <EditorFieldGroup>
+              <EditorField
+                width={15}
```

Comment:
The `width` prop here is setting a min-width, not a fixed one. So fields with these will only have a min width but will expand to accommodate the size of the selection. 
One way to set a fixed width would be to pass a className="width-30" to the component, but I didn't think that was necessary. I think it's a cool feature to have it adjustable (assuming the users don't have a database name of 350 characters )